File: LSL/liblsl-Generic/examples/SendData.py
import liblsl
import random
import time

# first create a new stream info (here we set the name to BioSemi, the content-type to EEG, 8 channels, 100 Hz, and float-valued data)
# The last value would be the serial number of the device or some other more or less locally unique identifier for the stream as far as available (you could also omit it but interrupted connections wouldn't auto-recover).
info = liblsl.stream_info('BioSemi','EEG',8,100,liblsl.cf_float32,'dsffwerwer');

# next make an outlet
outlet = liblsl.stream_outlet(info)

print("now sending data...")
while True:
	# make a new random 8-channel sample; this is converted into a liblsl.vectorf (the data type that is expected by push_sample)
	mysample = liblsl.vectorf([random.random(),random.random(),random.random(),random.random(),random.random(),random.random(),random.random(),random.random()])
	# now send it and wait for a bit
	outlet.push_sample(mysample)
	time.sleep(0.01)

File: LSL/liblsl-Java/src/examples/SendData.java
package examples;
import edu.ucsd.sccn.LSL;
import java.io.IOException;

public class SendData {
    public static void main(String[] args) throws IOException, InterruptedException  {
        System.out.println("Creating a new StreamInfo...");
        LSL.StreamInfo info = new LSL.StreamInfo("BioSemi","EEG",8,100,LSL.ChannelFormat.float32,"myuid324457");

        System.out.println("Creating an outlet...");
        LSL.StreamOutlet outlet = new LSL.StreamOutlet(info);
        
        System.out.println("Sending data...");
        float[] sample = new float[8];
        for (int t=0;t<100000;t++) {
            for (int k=0;k<8;k++)
                sample[k] = (float)Math.random()*50-25;
            outlet.push_sample(sample);
            Thread.sleep(10);
        }
        
        outlet.close();
        info.destroy();
    }
}